proto/kvrpcpb.proto

```diff
@@ -297,10 +297,11 @@ message CheckTxnStatusRequest {
     bool rollback_if_not_exist = 6;
 }
 
-enum RollbackReason {
-    NoReason = 0;
-    TTLExpire = 1;
-    LockNotExist = 2;
+enum Action {
+    Rollbacked = 0;
```

Comment:
We already know if it is rollbacked by lock_ttl and commit_version.
The default value should be NoAction.